SingularityUI/app/components/logs/FileNotFound.jsx

```diff
@@ -1,8 +1,36 @@
-import React from 'react';
+import React, {PropTypes} from 'react';
+
+const RENAMED_FILES = {
+  'service.log': 'tail_of_finished_service.log'
```

Comment:
Yeah you need to set your env variables correctly when you build. You can look at the jenkins configs to see the correct values.